### DISCUSSION TITLE
[Feature] Add ability to share and download multiple selected assets

### DISCUSSION THREAD

**Discussion by maxbrueckl** [immich-app/immich#3922] [Feature Request]
### The feature

If I select multiple assets in the Android App, I would like to have the ability to share and download the selected assets in bulk.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by localweights** [comment]
Came to request the same, but for iOS. We can share multiples, so just need download multiple added.

**Comment by maxbrueckl** [comment]
This is now possible in the latest version of the app on android.
